File: assignments_nilesh_sir/java_assignments/Assignment_Bank/Banking/Loan.java
package Banking;

public abstract class Loan {
    private double principle;
    private int period;

    public double getPrinciple() {
        return principle;
    }

    public void setPrinciple(double amount) {
        principle = amount;
    }

    public int getPeriod() {
        return period;
    }

    public void setPeriod(int p) {
        period = p;
    }

    public abstract double getRate();

    public double getEMI() {
        return principle * (1 + getRate() * period / 100) / (12 * period);
    }
}

File: Dotnet/04-05-24/Assignment_3/BankLib/Loan.cs
namespace BankLib{

public abstract class Loan{
    public double Principle {get;set;}

    public int Period {get; set;}

    public abstract double GetRate();

    public double GetEmi(){
        return Principle * (1 + GetRate() * Period / 100)/(12 * Period);
    }
}
}